### PR TITLE
added rating & toast

### PR DESCRIPTION
**Pull request by sanjay-xdr**
added rating and toast
rating demo :- https://loving-mestorf-0a3389.netlify.app/component/rating/rating
toast demo:-  https://loving-mestorf-0a3389.netlify.app/component/toast/toast

### CODE REVIEW

**Comment by Roshannair24** [comment]
Looks good to merge

**Review by Roshannair24 — APPROVED** [review]

File: Component/Rating/rating.html [hunk 1]
```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (vidhi-thakur) [inline Component/Rating/rating.html:6]: can add one more meta tag yo add your name as an author 

ex: image

File: Component/Toast/toast.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+.code-container {
+  width: 100%;
+}
+
+.btn {
+  height: 5rem;
+  width: 10rem;
+  font-size: 1.8rem;
+  margin: 2rem;
+  padding: 4px 6px;
```
Comment (vidhi-thakur) [inline Component/Toast/toast.css:12]: please use `rem` here

File: Component/Toast/toast.css [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  top: 10px;
+  right: 10px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+}
+
+.toast {
+  font-size: 2.5rem;
+  border-radius: 5px;
```
Comment (vidhi-thakur) [inline Component/Toast/toast.css:39]: could use `rem` here as well, kindly make the units consistent and prioritize using `rem`. Please update it on other places as well

File: Component/Toast/toast.css [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  padding: 1rem 2rem;
+  margin: 0.5rem;
+  width: 30rem;
+}
+
+.toast-danger {
+  background-color: red;
```
Comment (vidhi-thakur) [inline Component/Toast/toast.css:46]: this could be a variable

File: Component/Toast/toast.css [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+  color: #fff;
+}
+
+.toast-success {
+  background-color: green;
```
Comment (vidhi-thakur) [inline Component/Toast/toast.css:51]: same as above, a variable for `background-color` could be used here

**Review by vidhi-thakur — APPROVED** [review]
added comments for minor improvements

File: Component/Toast/toast.css (hunk 5 above)
Comment (sanjay-xdr) [reply Component/Toast/toast.css:51]: yes sure

**Pull request merged**